Libraries/LibWeb/CSS/FontFaceSet.cpp

```diff
@@ -175,7 +175,7 @@ static WebIDL::ExceptionOr<GC::Ref<JS::Set>> find_matching_font_faces(JS::Realm&
 {
     // 1. Parse font using the CSS value syntax of the font property. If a syntax error occurs, return a syntax error.
     auto property = parse_css_value(CSS::Parser::ParsingParams(), font, PropertyID::Font);
-    if (!property)
+    if (!property || !property->is_shorthand())
```

Comment:
This makes the next check redundant. Not sure what a nice way of dealing with that is though.

Reply:
Oh true, I've replaced the redundant check with a comment